datasets/persian_ner/README.md

```diff
@@ -69,7 +69,466 @@ The dataset includes 250,015 tokens and 7,682 Persian sentences in total. It is
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1,
+  1],
+ 'tokens': ['به',
+  'عنوان',
+  'مثال',
+  'وقتی',
+  'نشریات',
+  'مدافع',
+  'اصول',
+  'و',
+  'ارزشها',
+  'و',
+  'منادی',
+  'انقلاب',
+  'و',
+  'اسلام',
+  'در',
+  'بالاترین',
+  'درجه',
+  '،',
+  'اولین',
+  'و',
+  'درشت ترین',
+  'تیتر',
+  'نشریه',
+  'خود',
+  'را',
+  'در',
+  'صدر',
+  'صفحه',
+  'نخستین',
+  '،',
+  'به',
+  'تکذیب',
+  'اظهارات',
+  'و',
+  'نظریات',
+  'مشاور',
+  'رئیس جمهور',
+  'با',
+  'همین',
+  'ترکیب',
+  'عبارتی',
+  'و',
+  'البته',
+  'از',
+  'قول',
+  'دیگران',
+  'اختصاص',
+  'می دهند',
+  '،',
+  'آیا',
+  'در',
+  'موارد',
+  'مشابه',
+  'نیز',
+  'هر',
+  'گاه',
+  'خبر',
+  'تکذیب',
+  'متوجه',
+  'و',
+  'معطوف',
+  'به',
+  'شخصی',
+  'باشد',
+  'كه',
+  'در',
+  'زمره',
+  'مشاوران',
+  'يك',
+  'مقام',
+  'بلندمرتبه ی',
+  'دیگر',
+  'است',
+  '،',
+  'خبر',
+  'را',
+  'عینا',
+  'به',
+  'همین',
+  'درشتی',
+  'و',
+  'با',
+  'همین',
+  'ترکیب',
+  'عبارتی',
+  'در',
+  'صدر',
+  'صفحه',
+  'نخست',
+  'به',
+  'چاپ',
+  'می رساند',
+  'و',
+  'در',
+  'آن',
+  'مورد',
+  'هم',
+  'به',
+  'جای',
+  'ذکر',
+  'نام',
+  'يا',
+  'عضویت',
+  'آن',
+  'شخص',
+  'در',
+  'گروه',
+  'و',
+  'کمیتهی',
+  'خاص',
+  'صرفا',
+  'بر',
+  'روی',
+  'عنوان',
+  'مشاور',
+  'فلان',
+  'مسئول',
+  'بلندمرتبه',
+  'تأکید',
+  'میکنند',
+  '؟',
+  'اگر',
+  'نشریه',
+  'دیگری',
+  'چنین',
+  'کند',
+  '،',
+  'شدیدا',
+  'مورد',
+  'انتقاد',
+  'آنان',
+  'قرار',
+  'نمیگیرد',
+  'كه',
+  'صورت',
+  'خبر',
+  'را',
+  'بهانه',
+  'کرده',
+  'و',
+  'موضوع',
+  'دیگری',
+  'را',
+  'هدف',
+  'قرار',
+  'داده',
+  'است',
+  '؟',
+  'بااینهمه',
+  '،',
+  'بازهم',
+  'از',
+  'قضاوت',
+  'قطعی',
+  'پرهیز',
+  'کرده',
+  '،',
+  'میگوئیم',
+  'فقط',
+  'نتیجه',
+  'مقایسه',
+  'و',
+  'تطبیق',
+  'در',
+  'يك',
+  'وضعیت',
+  'جامعالاطراف',
+  'نشان',
+  'خواهد',
+  'داد',
+  'كه',
+  'رسانههای',
+  'جبههی',
+  'منتقد',
+  'و',
+  'مخالف',
+  'رئیسجمهور',
+  'و',
+  'دولت',
+  'در',
+  'مقام',
+  'نقادی',
+  'واقعا',
+  'یکسان',
+  'و',
+  'بدون',
+  'تبعیض',
+  'عمل',
+  'کردهاند',
+  'يا',
+  'خدایناکرده',
+  'ولو',
+  'در',
+  'عالم',
+  'فرض',
+  'به',
+  'عنوان',
+  'مثال',
+  'غیر',
+  'دولت',
+  'و',
+  'غیر',
+  'رئیسجمهور',
+  'را',
+  'با',
+  'نوازشهای',
+  'لسانی',
+  'و',
+  'قلمی',
+  'نقد',
+  'کردهاند',
+  'اما',
+  'دولت',
+  'و',
+  'رئیسجمهور',
+  'را',
+  'با',
+  'خمپارههای',
+  'کلامی',
+  'و',
+  'نوشتاری',
+  'هدف',
+  'نقادیهای',
+  'موسوم',
+  'به',
+  'نقد',
+  'خیرخواهانه',
+  'قرار',
+  'دادهاند',
+  '.']}
```

Comment:
A few comments:
- You can keep only the first 5 tokens or so, it will be more readable this way
- It would be nice to enclose this in a code block (delimited by three backticks ` )
- The ner_labels looks weird, why is it always 1? If this is a real sample from the dataset there's probably something wrong

Reply:
Resolved the first two points and on the third point, I think the first sentence of the train dataset just has 0 tags. This isn't a recurring issue so I just copied over the 5 examples from the second sentence.